### PR TITLE
Weapons Update: Part 1 Guns n' Improvised Guns.

### PR DESCRIPTION
**Pull request by KathrinBailey**
## Media
image

## About The Pull Request
• Reworks the improvised shotgun into a break-action shotgun. It is now a subtype of rifle and using its action, so you break it open and have to close it. It also has its own sprites. It's a tad more expensive to craft. Requires two hands to fire now.
• Introduces the improvised rifle with its own new sprite. A single-shot 7.62 bolt action rifle. Requires two hands to fire.
• Introduces the revolving rifles. There is a craft-able improvised version, a normal version and a gold version that fires .45 instead of the usual .38 that the improvised and normal versions fire. They are bulky and require two hands to fire.
• Adds inhands sprites for the shotgun, rifle, revolving rifle and gold revolving rifle.
• Adds the .32 ammo, 15 damage.
• Adds rubber bullet versions of some existing ammo, such as .45 and 4.6x30mm.
• Makes the new bullets craft-able in hacked autolathes. Just the bullets. Adds an **empty** .32 SMG magazine too.
• Adds a bunch of currently adminspawn only weapons:
   • CFA Ruby - 8rnd .45 pistol.
   • CFA Snub - 12rnd 4.6x30mm pistol.
   • CFA Wildcat - 30rnd .32 SMG. It's pretty much a three round burst MP5.
   • Canta Rifle - A semi-automatic 5.56 rifle with a 10rnd magazine.

## Why It's Good For The Game
**Improvised Weapons:**
With TG wounds, some of this stuff needs toning down. The shotgun had too much utility. Walk up to someone with two improvised shotguns, bam. You're gone even if you get no wounds. If you're still standing, then you're about two welder love taps from being out. Some of these weapons need plasteel which you have in insane abundance if you have maint access or you can follow someone into maint. Oh, you probably have ORM access anyway.
The rifle was something existing on Cit and now it's just being brought here. You don't have the different ammo types like the shotgun. It could be made more expensive if there's a desire for it. It's styled after a Short Magazine Lee Enfield Mk.III
The revolving rifle is something different. It's the only improvised gun that has a capacity greater than a single bullet. It fills a niche. You can't store this in your bag (bulky) nor shorten it. It fires slowly so you can't instantly magdump into someone. It uses .38 like the detective's revolver. The higher capacity should help make gunplay more interesting for folks.

**Themed Weapons:**
The four weapons are themed around coming from one alien planet in a Great War-like period.
The **CFA Ruby** is an old standard military service sidearm. It's styled after a M1911.  It uses .45 (30 Brute, 8rnd).
The **CFA Snub** is a compact military sidearm. It's styled after a FN Model 1910 and P83 Wanad. It's meant to be a lightweight easily concealed weapon. It uses 4.6x30mm (20 brute, 12rnd).
The **Canta Rifle** is an even older military rifle. It's based on the M1 Garand. You can zoom with it like a sniper, but you don't zoom as much since it's iron sights. You're kinda just focusing with it. It uses 5.56 rounds. (35 Brute, 10rnd).
The **CFA Wildcat**. This is the near end of war weapon. It's actually just a MP5. It's a very iconic weapon, so people get to use it. It doesn't use 9mm because that'd make the weapon nukie-tier. High magazine capacity and low damage makes this weapon not absurd and very very very fun to use. It uses .32, which will be used more in the future. (15 Brute, 30rnd).
So, why is it good? I'm not sure what to do with these yet, so they're being left for adminbus and TC trades. They're all fairly well balanced and usable. They aren't ridiculously overpowered, just be aware of what they are and what they do. If you ever want to do something with World War vibes, here's extra variety. 

**In Summary:**
These weapons should make gunplay more fun and interesting. Also the sprites are better. Plus, you can't whip out a round removal cannon with three minutes of effort anymore.

## Changelog
:cl: Floof Ball#0798
add: Adds four currently adminspawn only weapons which are all World War inspired.
add: Adds the improvised rifle and improvised revolving rifle.
tweak: Reworks the improvised shotgun. It's now a break-action shotgun. It has new sprites too.
/:cl:

### CODE REVIEW

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by Gandalf2k15** [comment]
Checks complete, we'll merge this once travis completes.
